rerun_js/web-viewer/index.ts

```diff
@@ -195,7 +195,7 @@ export class WebViewer {
 
     this.#handle = new WebHandle_class({ ...options, fullscreen });
     try {
-      await this.#handle.start(this.#canvas.id);
```

Comment:
We should also remove all the `this.#id` shenanigans here, seeing as it's no longer used